internal/worker/domainservices/manifold.go

```diff
@@ -261,7 +261,7 @@ func NewProviderTrackerModelDomainServices(
 	modelUUID coremodel.UUID,
 	dbGetter changestream.WatchableDBGetter,
 	providerFactory providertracker.ProviderFactory,
-	objectStore objectstore.ModelObjectStoreGetter,
+	modelObjectStore objectstore.ModelObjectStoreGetter,
```

Comment:
Nit: should this be modelObjectStoreGetter to standardise with the above and the type?

Reply:
Yep good pickup.